src/app/app.component.ts

```diff
@@ -16,6 +18,7 @@ import { LogoutBasePageComponent } from '@shared/classes/logout-base-page';
 import { LoadAppVersion, AppResumed, AppSuspended } from '@store/app-info/app-info.actions';
 import { selectLogoutEnabled } from '@store/app-config/app-config.selectors';
 
+
```

Comment:
Double line here should be an ESLint failure (I think)

Reply:
weird, looks like its waiting to be commited